packages/react-dom/src/__tests__/ReactDOM-test.js

```diff
@@ -377,6 +377,26 @@ describe('ReactDOM', () => {
     }
   });
 
+  it('should not throw a TypeError when reading stateNode on ReactCurrentOwner', () => {
+    class Component extends React.Component {
+      render() {
+        return <div />;
+      }
+    }
+
+    const App = () => ReactDOM.findDOMNode(Component);
+    const container = document.createElement('div');
+    const stateNodeErr = new TypeError(
+      "Cannot read property '_warnedAboutRefsInRender' of null",
+    );
+
+    if (__DEV__) {
+      expect(() => {
+        ReactDOM.render(<App />, container);
+      }).not.toThrow(stateNodeErr);
+    }
```

Comment:
It's enough to replace this whole block with just `ReactDOM.render(<App />, container)`. If it did throw, the test would fail anyway. You can remove the error declaration too.